x-pack/legacy/plugins/siem/public/components/page/hosts/authentications_table/index.tsx

```diff
@@ -4,6 +4,8 @@
  * you may not use this file except in compliance with the Elastic License.
  */
 
+/* eslint-disable react/display-name */
+
```

Comment:
Just noting the consistency of naming our connected components. Seems it's hit or miss on if we set the `displayName` after calling `connect()`. Here in `AuthenticationTable` we do not set `displayName` after connecting, but we do for `UncommonProcessTable`.  The linter rule does not enforce it for connected components, so we'll need to keep an eye out in reviews (or perhaps see if the updated `react-redux` can handle setting the displayName?).